source/unified-test-format/unified-test-format.rst

```diff
@@ -1177,7 +1177,7 @@ The structure of each object is as follows:
   `CMAP
   <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#events>`__
   events, and ``sdam`` for `SDAM
-  <../server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#events>`__
+  <../server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst#events>`__
```

Comment:
@BorisDog should we remain all this file references too?

Reply:
The intention is to rename `server-discovery-and-monitoring-monitoring.rst `to `server-discovery-and-monitoring-logging-and-monitoring.rst` as last step of the PR. So the references are updated to reflect that.

Reply:
I have failed to force github to recognize the renaming within the PR (need to ensure it's recognized in git history).
The naming follows [command-logging-and-monitoring](https://github.com/mongodb/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst) change where `logging` was added to `monitoring`. I see a value in having consistent naming across the spec, as long as it does not introduce unnecessary overhead.

Side note: For me `monitoring-monitoring` suffix has been a bit confusing, so longer name with `logging` is bit clearer :)

Reply:
Reverted all renaming residue. Due to non-trivial commit history in this PR, renaming has to be considered in a separate ticket.